docs/advanced/peering/peering-via-cr.md

```diff
@@ -0,0 +1,451 @@
+    liqo.io/remote-cluster-id: <PROVIDER_CLUSTER_ID>
+  annotations:
+    liqo.io/remote-tenant-namespace: <PROVIDER_TENANT_NAMESPACE>
+  name: cplane-secret
+  namespace: <TENANT_NAMESPACE>
+```
+
+Once you create this secret, the `liqo-crd-replicator` starts the replication of the resources, and enables the creation of `ResourceSlice` resources targetting the provider cluster. This allows the consumer to start negotiating resources with the provider cluster.
+
+After the secret creation, the logs of the `liqo-crd-replicator` should contain the following records:
+
+```text
+ k logs -n liqo liqo-crd-replicator-68f9f55dfc-bc4l8 --tail 5
+
+I1107 10:05:29.249872       1 crdReplicator-operator.go:94] Processing Secret "liqo-tenant-cl02/kubeconfig-controlplane-cl02"
+I1107 10:05:29.254977       1 reflector.go:90] [cl02] Starting reflection towards remote cluster
+I1107 10:05:29.255001       1 reflector.go:131] [cl02] Starting reflection of offloading.liqo.io/v1beta1, Resource=namespacemaps
+I1107 10:05:29.255035       1 reflector.go:131] [cl02] Starting reflection of authentication.liqo.io/v1beta1, Resource=resourceslices
+I1107 10:05:29.355741       1 reflector.go:163] [cl02] Reflection of authentication.liqo.io/v1beta1, Resource=resourceslices correctly started
+```
+
+### Summary of authentication configuration
+
+To sum up, to set up the authentication, **on the provider cluster** you will need to:
+
+- Create the credentials to be used by the consumer
+- Bind the credentials to the `liqo-remote-controlplane` role
+- Create a `Tenant` resource for the consumer
+
+While, **on the consumer side**:
+
+- Create a new secret containing the kubeconfig with the credentials to access to the provider cluster
+
+## Declarative configuration of namespace offloading
+
+While offloading is independent from the network, which means that it is possible to negotiate resources and configure a namespace offloading without the inter-cluster network enabled, **a [working authentication configuration](#declarative-configuration-of-clusters-authentication) is a pre-requisite to enable offloading**.
+
+### Ask for resources: configure a ResourceSlice (consumer cluster)
+
+The `ResourceSlice` resource is the CR that defines the computational resources requested by the consumer to the provider cluster.
+It should be created on the tenant namespace of the consumer cluster, and it is automatically forwarded to the provider cluster, which can accept or reject it.
+
+The following is an example of `ResourceSlice`:
+
+```yaml
+apiVersion: authentication.liqo.io/v1beta1
+kind: ResourceSlice
+metadata:
+  annotations:
+    liqo.io/create-virtual-node: "true"
+  creationTimestamp: null
+  labels:
+    liqo.io/remote-cluster-id: <PROVIDER_CLUSTER_ID>
+    liqo.io/remoteID: <PROVIDER_CLUSTER_ID>
+  name: test
+  namespace: <CONSUMER_TENANT_NAMESPACE>
+spec:
+  class: default
+  providerClusterID: <PROVIDER_CLUSTER_ID>
+  resources:
+    cpu: 20
+    ram  128Gi
```

Comment:
Hello @claudiolor .

I suggest the following modifications on this yaml:

resources:
cpu: '20'
ram : 128Gi
pods: '110'

As reported into this slack [thread](https://liqo-io.slack.com/archives/C014Z7F25KP/p1732027216878499).

Sorry if I'm just commenting, but I don't have permissions for editing files.

Thanks.

Reply:
Hi Francesco, if the `pods` resource is missing, then the default value is used, so I would not add it in this part of the documentation for simplicity.
If you are experiencing any issue when you don't provide the resources, I suggest to open an issue in this repository with the details of your problem and how to reproduce it, if it is actually a bug, then we will take care of fixing it.

Reply:
Ops, thanks for let me notice it :)